usage_examples/README.md

```diff
@@ -97,27 +97,18 @@ You can check sample tasks for Named Entity Recognition and text categorization
 in the `spacy_llm/tasks/` directory. We also recommend checking out the
 `spacy.NoOp.v1` task for a barebones implementation to pattern your task from.
 
-## Using LangChain, MiniChain and other integrated third-party prompting libraries
+## Using LangChain and other integrated third-party prompting libraries
 
 `spacy-llm` integrates bindings to a number of libraries centered on prompt management and LLM usage to allow users
-to leverage their functionality in their spaCy workflows. This currently includes
-
-- [LangChain](https://github.com/hwchase17/langchain)
-- [MiniChain](https://github.com/srush/MiniChain)
+to leverage their functionality in their spaCy workflows. This currently includes [LangChain](https://github.com/hwchase17/langchain)
 
```

Comment:
I would rephrase and say that `spacy-llm` allows integrations with external libraries, and that LangChain is one built-in example.